content/api/commands/intercept.md

```diff
@@ -170,116 +94,153 @@ The `routeHandler` defines what will happen with a request if the [routeMatcher]
 
 ## Examples
 
-### Matching URL
+<!-- TODO DX-86 Add granular headings for code snippets/examples for link-ability -->
+<!-- TODO DX-88 Add a routeHandler options section that includes examples for each option -->
+
+### Matching a URL
 
-You can provide the entire URL to match
+You can provide the exact URL to match, but typically pattern matching is more effective and a glob pattern will likely be the simpler of the two to implement. See the Glob Pattern Matching section.
 
 ```js
-// will match any request that exactly matches the URL
-//   matches GET https://prod.cypress.io/users
-//   won't match GET https://staging.cypress.io/users
+// match any request that exactly matches the URL
 cy.intercept('https://prod.cypress.io/users')
+
+// match any request that satisfies a glob pattern
+cy.intercept('**/users?_limit=+(3|5)')
+
+// match any request that satisfies a regex pattern
+cy.intercept(/\/users\?_limit=(3|5)$/)
 ```
 
-You can provide a [`minimatch`](/api/utilities/minimatch) pattern
+<!-- TODO move this section -->
+### Glob Pattern Matching URLs
+
+Providing an exact URL to match can be too restrictive. For example, what if you wanted to run your tests on a different host? 
+
+```js
+// match any request that exactly matches the URL
+cy.intercept('https://prod.cypress.io/users')
+// matches this: https://prod.cypress.io/users
+// ...but not this: https://staging.cypress.io/users
+// ...or this: http://localhost/users
+```
+Glob pattern matching provides the necessary flexibility:
+
+```js
+cy.intercept('**/users')
+// matches all of these:
+// https://prod.cypress.io/users
+// https://staging.cypress.io/users
+// http://localhost/users
 
-```javascript
-// will match any HTTP method to urls that end with 3 or 5
 cy.intercept('**/users?_limit=+(3|5)')
+// matches all of these:
+// https://prod.cypress.io/users?_limit=3
+// http://localhost/users?_limit=5
+
```

Comment:
I decided to sell glob pattern matching here. I'm likely going to move the bulk of this further down and link to it from here.